js/main.js

```diff
@@ -36,22 +61,40 @@ function handleInput(event) {
 
 function handleSubmit(event) {
   event.preventDefault();
-  var newObj = {
-    title: $createForm.elements.title.value,
-    imageURL: $createForm.elements.photourl.value,
-    notes: $createForm.elements.notes.value
-  };
-  newObj.entryId = data.nextEntryId;
-  data.nextEntryId++;
-  data.entries.unshift(newObj);
-  $ul.prepend(renderEntry(newObj));
+  var $liList = document.querySelectorAll('li');
+  if (data.editing === null) {
+    var newObj = {
+      title: $createForm.elements.title.value,
+      imageURL: $createForm.elements.photourl.value,
+      notes: $createForm.elements.notes.value
+    };
+    newObj.entryId = data.nextEntryId;
+    data.nextEntryId++;
+    data.entries.unshift(newObj);
+    $ul.prepend(renderEntry(newObj));
+    data.editing = null;
+  } else {
+    for (var i = 0; i < $liList.length; i++) {
+      var updateObj = {
+        title: $createForm.elements.title.value,
+        imageURL: $createForm.elements.photourl.value,
+        notes: $createForm.elements.notes.value,
+        entryId: data.editing.entryId
+      };
+      $liList[i].replaceWith(renderEntry(updateObj));
+    }
```

Comment:
Wouldn't this make every entry in your page the same edited entry? Have you tested this with multiple entries on the screen?